modules/tinymce/src/themes/silver/main/ts/ui/throbber/Throbber.ts

```diff
@@ -62,19 +54,59 @@ const renderThrobber = (spec): AlloySpec => ({
     }
   },
   behaviours: Behaviour.derive([
-    Replacing.config({})
+    Replacing.config({}),
+    Blocking.config({
+      focus: false
+    }),
+    Composing.config({
+      find: (comp) => Arr.head(comp.components())
+    })
   ]),
   components: [ ]
 });
 
+const focusBusyComponent = (throbber: AlloyComponent): void => {
+  Composing.getCurrent(throbber).each((comp) => {
+    Keying.focusIn(comp);
+  });
+};
```

Comment:
```suggestion
const focusBusyComponent = (throbber: AlloyComponent): void =>
  Composing.getCurrent(throbber).each(Keying.focusIn);
```
This can be ETA reduced and can be flattened